### ISSUE TITLE
Frequent tab crashes in `firefox-wayland`

### ISSUE THREAD

**Issue by mhwombat** [NixOS/nixpkgs#147025]
### Describe the bug
Firefox tabs crash frequently in wayland. This happens with different sites, but I can reproduce it regularly with YouTube.

### Steps To Reproduce
1. Add `firefox-wayland` to `environment.systemPackages` in NixOS configuration.
2. Ensure the following environment variable is set: MOZ_ENABLE_WAYLAND=1
3. Rebuild NixOS: `nixos-rebuild switch --upgrade`.
4. Start a wayland compositor (I have reproduced this on Sway and Plasma)
5. Start Firefox in safe mode to disable add-ons: firefox --safe-mode.
6. Navigate to YouTube (https://www.youtube.com/)
7. Hover your mouse over one of the video thumbnails.

This almost always causes the tab to crash for me, but occasionally I have to hover over a few different video thumbnails to reproduce the problem.

### Expected behavior
I expect tabs not to crash.

### Screenshots
A screenshot probably isn't helpful.

### Additional context

I verified that Firefox is running under Wayland rather than Xwayland. (My sway configuration show this in the client titlebar.)

```
$ firefox -v
Mozilla Firefox 94.0.1
```
### Notify maintainers
@taku0 
@mweinelt 

### Metadata

```console
$ nix-shell -p nix-info --run "nix-info -m"
 - system: `"x86_64-linux"`
 - host os: `Linux 5.10.79, NixOS, 21.11 (Porcupine)`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.5pre20211007_844dd90`
 - channels(amy): `""`
 - channels(root): `"nixos-21.11pre332033.715f6341195"`
 - nixpkgs: `/nix/var/nix/profiles/per-user/root/channels/nixos`
```

Maintainer information:
```yaml
# a list of nixpkgs attributes affected by the problem
attribute:
# a list of nixos modules affected by the problem
module:
```

**Comment by mhwombat** [comment]
Troubleshooting information from `about:support`.

```
{
  "application": {
    "name": "Firefox",
    "osVersion": "Linux 5.10.79 #1-NixOS SMP Fri Nov 12 13:58:35 UTC 2021",
    "version": "94.0.1",
    "buildID": "20211116142757",
    "distributionID": "",
    "userAgent": "Mozilla/5.0 (X11; Linux x86_64; rv:94.0) Gecko/20100101 Firefox/94.0",
    "safeMode": true,
    "supportURL": "https://support.mozilla.org/1/firefox/94.0.1/Linux/en-US/",
    "osTheme": "Adwaita / Adwaita",
    "numTotalWindows": 7,
    "numFissionWindows": 0,
    "numRemoteWindows": 7,
    "fissionAutoStart": false,
    "fissionDecisionStatus": "disabledBySafeMode",
    "remoteAutoStart": true,
    "policiesStatus": 0,
    "keyLocationServiceGoogleFound": true,
    "keySafebrowsingGoogleFound": true,
    "keyMozillaFound": false
  },
  "securitySoftware": {
    "registeredAntiVirus": "",
    "registeredAntiSpyware": "",
    "registeredFirewall": ""
  },
  "processes": {
    "remoteTypes": {
      "web": 6,
      "extension": 1,
      "privilegedabout": 1,
      "prealloc": 1
    },
    "maxWebContentProcesses": 8
  },
  "environmentVariables": {
    "GDK_PIXBUF_MODULE_FILE": "/nix/store/1c5yr5g4lzrf4macxw1pvglqi9sn8kb4-librsvg-2.52.3/lib/gdk-pixbuf-2.0/2.10.0/loaders.cache",
    "MOZ_LEGACY_PROFILES": "1",
    "MOZ_ALLOW_DOWNGRADE": "1",
    "MOZ_APP_LAUNCHER": "firefox",
    "DISPLAY": ":1",
    "MOZ_ENABLE_WAYLAND": "1",
    "MOZ_PLUGIN_PATH": "/home/amy/.nix-profile/lib/mozilla/plugins:/etc/profiles/per-user/amy/lib/mozilla/plugins:/nix/var/nix/profiles/default/lib/mozilla/plugins:/run/current-system/sw/lib/mozilla/plugins",
    "MOZ_SYSTEM_DIR": "/nix/store/05ijja81zfbn0s3lc9326fw7sdx647dn-firefox-94.0.1/lib/mozilla",
    "MOZ_ASSUME_USER_NS": "1",
    "MOZ_LAUNCHED_CHILD": "",
    "MOZ_APP_SILENT_START": "",
    "XRE_PROFILE_PATH": "",
    "XRE_PROFILE_LOCAL_PATH": "",
    "XRE_START_OFFLINE": "",
    "XRE_BINARY_PATH": "",
    "XRE_RESTARTED_BY_PROFILE_MANAGER": "",
    "MOZ_APP_RESTART": "1",
    "MOZ_SAFE_MODE_RESTART": ""
  },
  "modifiedPreferences": {
    "accessibility.typeaheadfind.flashBar": 0,
    "browser.cache.disk.amount_written": 1834436,
    "browser.cache.disk.capacity": 1048576,
    "browser.cache.disk.filesystem_reported": 1,
    "browser.cache.disk.hashstats_reported": 1,
    "browser.contentblocking.category": "standard",
    "browser.search.region": "IE",
    "browser.sessionstore.upgradeBackup.latestBuildID": "20211116142757",
    "browser.startup.homepage_override.buildID": "20211116142757",
    "browser.startup.homepage_override.mstone": "94.0.1",
    "browser.startup.page": 3,
    "browser.urlbar.matchBuckets": "general:5,suggestion:Infinity",
    "browser.urlbar.placeholderName": "DuckDuckGo",
    "browser.urlbar.resultBuckets": "{\"children\":[{\"maxResultCount\":1,\"children\":[{\"group\":\"heuristicTest\"},{\"group\":\"heuristicExtension\"},{\"group\":\"heuristicSearchTip\"},{\"group\":\"heuristicOmnibox\"},{\"group\":\"heuristicUnifiedComplete\"},{\"group\":\"heuristicAutofill\"},{\"group\":\"heuristicTokenAliasEngine\"},{\"group\":\"heuristicFallback\"}]},{\"group\":\"extension\",\"maxResultCount\":5},{\"flexChildren\":true,\"children\":[{\"group\":\"general\",\"flex\":2},{\"flexChildren\":true,\"children\":[{\"flex\":2,\"group\":\"formHistory\"},{\"flex\":4,\"group\":\"remoteSuggestion\"},{\"flex\":0,\"group\":\"tailSuggestion\"}],\"flex\":1}]}]}",
    "browser.urlbar.resultGroups": "{\"children\":[{\"maxResultCount\":1,\"children\":[{\"group\":\"heuristicTest\"},{\"group\":\"heuristicExtension\"},{\"group\":\"heuristicSearchTip\"},{\"group\":\"heuristicOmnibox\"},{\"group\":\"heuristicEngineAlias\"},{\"group\":\"heuristicBookmarkKeyword\"},{\"group\":\"heuristicAutofill\"},{\"group\":\"heuristicPreloaded\"},{\"group\":\"heuristicTokenAliasEngine\"},{\"group\":\"heuristicFallback\"}]},{\"group\":\"extension\",\"availableSpan\":5},{\"flexChildren\":true,\"children\":[{\"group\":\"generalParent\",\"children\":[{\"availableSpan\":3,\"group\":\"inputHistory\"},{\"flexChildren\":true,\"children\":[{\"flex\":1,\"group\":\"remoteTab\"},{\"flex\":2,\"group\":\"general\"},{\"flex\":2,\"group\":\"aboutPages\"},{\"flex\":1,\"group\":\"preloaded\"}]},{\"group\":\"inputHistory\"}],\"flex\":2},{\"children\":[{\"flexChildren\":true,\"children\":[{\"flex\":2,\"group\":\"formHistory\"},{\"flex\":4,\"group\":\"remoteSuggestion\"}]},{\"group\":\"tailSuggestion\"}],\"flex\":1}]}]}",
    "browser.urlbar.showSearchSuggestionsFirst": false,
    "browser.urlbar.tabToSearch.onboard.interactionsLeft": 0,
    "browser.urlbar.tipShownCount.searchTip_onboard": 4,
    "browser.urlbar.tipShownCount.searchTip_redirect": 3,
    "browser.urlbar.tipShownCount.tabToSearch": 20,
    "doh-rollout.balrog-migration-done": true,
    "doh-rollout.doneFirstRun": true,
    "doh-rollout.home-region": "IE",
    "dom.push.userAgentID": "95299b874a034120ad55ef34b30be16e",
    "extensions.lastAppVersion": "94.0.1",
    "idle.lastDailyNotification": 1637543549,
    "media.gmp-gmpopenh264.abi": "x86_64-gcc3",
    "media.gmp-gmpopenh264.lastUpdate": 1594229695,
    "media.gmp-gmpopenh264.version": "1.8.1.1",
    "media.gmp-manager.buildID": "20211116142757",
    "media.gmp-manager.lastCheck": 1637601318,
    "media.gmp.storage.version.observed": 1,
    "media.videocontrols.picture-in-picture.video-toggle.has-used": true,
    "network.dns.disablePrefetch": true,
    "network.http.speculative-parallel-limit": 0,
    "network.predictor.enabled": false,
    "network.prefetch-next": false,
    "network.trr.blocklist_cleanup_done": true,
    "places.database.lastMaintenance": 1637543552,
    "places.history.expiration.transient_current_max_pages": 148104,
    "privacy.sanitize.pending": "[{\"id\":\"newtab-container\",\"itemsToClear\":[],\"options\":{}}]",
    "privacy.purge_trackers.date_in_cookie_database": "0",
    "privacy.purge_trackers.last_purge": "1637543549550",
    "security.remote_settings.crlite_filters.checked": 1637593564,
    "security.remote_settings.intermediates.checked": 1637593564,
    "security.sandbox.content.tempDirSuffix": "7a6d0cc4-2bc6-46b1-ae46-bc36e27900b7",
    "security.sandbox.plugin.tempDirSuffix": "b5d95192-e504-41dd-9e79-441ef95166ac",
    "services.sync.declinedEngines": "",
    "services.sync.lastPing": 1637534419,
    "services.sync.lastSync": "Mon Nov 22 2021 17:59:53 GMT+0000 (Greenwich Mean Time)",
    "services.sync.engine.prefs.modified": false,
    "signon.importedFromSqlite": true,
    "signon.usage.hasEntry": true,
    "signon.usage.lastUsed": 1613568798,
    "storage.vacuum.last.index": 1,
    "storage.vacuum.last.places.sqlite": 1636851645
  },
  "lockedPreferences": {
    "fission.autostart.session": false
  },
  "printingPreferences": {
    "print.more-settings.open": true,
    "print.print_bgcolor": false,
    "print.print_bgimages": false,
    "print.print_duplex": 0,
    "print.print_evenpages": true,
    "print.print_margin_bottom": "0.5",
    "print.print_margin_left": "0.5",
    "print.print_margin_right": "0.5",
    "print.print_margin_top": "0.5",
    "print.print_oddpages": true,
    "print.print_orientation": 0,
    "print.print_page_delay": 50,
    "print.print_paper_data": 0,
    "print.print_paper_height": " 11.69",
    "print.print_paper_name": "iso_a4",
    "print.print_paper_size_unit": 0,
    "print.print_paper_width": "  8.27",
    "print.print_scaling": "  1.00",
    "print.print_shrink_to_fit": true,
    "print.print_to_file": false,
    "print.print_unwriteable_margin_bottom": 56,
    "print.print_unwriteable_margin_left": 25,
    "print.print_unwriteable_margin_right": 25,
    "print.print_unwriteable_margin_top": 25,
    "print.printer_HP_Envy_6032.print_bgcolor": false,
    "print.printer_HP_Envy_6032.print_bgimages": false,
    "print.printer_HP_Envy_6032.print_duplex": 1,
    "print.printer_HP_Envy_6032.print_edge_bottom": 0,
    "print.printer_HP_Envy_6032.print_edge_left": 0,
    "print.printer_HP_Envy_6032.print_edge_right": 0,
    "print.printer_HP_Envy_6032.print_edge_top": 0,
    "print.printer_HP_Envy_6032.print_footercenter": "",
    "print.printer_HP_Envy_6032.print_footerleft": "&PT",
    "print.printer_HP_Envy_6032.print_footerright": "&D",
    "print.printer_HP_Envy_6032.print_headercenter": "",
    "print.printer_HP_Envy_6032.print_headerleft": "&T",
    "print.printer_HP_Envy_6032.print_headerright": "&U",
    "print.printer_HP_Envy_6032.print_in_color": true,
    "print.printer_HP_Envy_6032.print_margin_bottom": "0.5",
    "print.printer_HP_Envy_6032.print_margin_left": "0.5",
    "print.printer_HP_Envy_6032.print_margin_right": "0.5",
    "print.printer_HP_Envy_6032.print_margin_top": "0.5",
    "print.printer_HP_Envy_6032.print_orientation": 0,
    "print.printer_HP_Envy_6032.print_page_delay": 50,
    "print.printer_HP_Envy_6032.print_paper_height": "11.6929",
    "print.printer_HP_Envy_6032.print_paper_id": "iso_a4_210x297mm",
    "print.printer_HP_Envy_6032.print_paper_size_unit": 0,
    "print.printer_HP_Envy_6032.print_paper_width": "8.26772",
    "print.printer_HP_Envy_6032.print_resolution": 0,
    "print.printer_HP_Envy_6032.print_reversed": false,
    "print.printer_HP_Envy_6032.print_scaling": "1",
    "print.printer_HP_Envy_6032.print_shrink_to_fit": true,
    "print.printer_HP_Envy_6032.print_to_file": false,
    "print.printer_HP_Envy_6032.print_unwriteable_margin_bottom": 12,
    "print.printer_HP_Envy_6032.print_unwriteable_margin_left": 12,
    "print.printer_HP_Envy_6032.print_unwriteable_margin_right": 12,
    "print.printer_HP_Envy_6032.print_unwriteable_margin_top": 12,
    "print.printer_Mozilla_Save_to_PDF.print_bgcolor": false,
    "print.printer_Mozilla_Save_to_PDF.print_bgimages": false,
    "print.printer_Mozilla_Save_to_PDF.print_duplex": 0,
    "print.printer_Mozilla_Save_to_PDF.print_edge_bottom": 0,
    "print.printer_Mozilla_Save_to_PDF.print_edge_left": 0,
    "print.printer_Mozilla_Save_to_PDF.print_edge_right": 0,
    "print.printer_Mozilla_Save_to_PDF.print_edge_top": 0,
    "print.printer_Mozilla_Save_to_PDF.print_footercenter": "",
    "print.printer_Mozilla_Save_to_PDF.print_footerleft": "&PT",
    "print.printer_Mozilla_Save_to_PDF.print_footerright": "&D",
    "print.printer_Mozilla_Save_to_PDF.print_headercenter": "",
    "print.printer_Mozilla_Save_to_PDF.print_headerleft": "&T",
    "print.printer_Mozilla_Save_to_PDF.print_headerright": "&U",
    "print.printer_Mozilla_Save_to_PDF.print_in_color": true,
    "print.printer_Mozilla_Save_to_PDF.print_margin_bottom": "0.5",
    "print.printer_Mozilla_Save_to_PDF.print_margin_left": "0.5",
    "print.printer_Mozilla_Save_to_PDF.print_margin_right": "0.5",
    "print.printer_Mozilla_Save_to_PDF.print_margin_top": "0.5",
    "print.printer_Mozilla_Save_to_PDF.print_orientation": 0,
    "print.printer_Mozilla_Save_to_PDF.print_page_delay": 50,
    "print.printer_Mozilla_Save_to_PDF.print_paper_height": "11.6929",
    "print.printer_Mozilla_Save_to_PDF.print_paper_id": "iso_a4",
    "print.printer_Mozilla_Save_to_PDF.print_paper_size_unit": 0,
    "print.printer_Mozilla_Save_to_PDF.print_paper_width": "8.26772",
    "print.printer_Mozilla_Save_to_PDF.print_reversed": false,
    "print.printer_Mozilla_Save_to_PDF.print_scaling": "1",
    "print.printer_Mozilla_Save_to_PDF.print_shrink_to_fit": true,
    "print.printer_Mozilla_Save_to_PDF.print_to_file": true,
    "print.printer_Mozilla_Save_to_PDF.print_unwriteable_margin_bottom": 0,
    "print.printer_Mozilla_Save_to_PDF.print_unwriteable_margin_left": 0,
    "print.printer_Mozilla_Save_to_PDF.print_unwriteable_margin_right": 0,
    "print.printer_Mozilla_Save_to_PDF.print_unwriteable_margin_top": 0,
    "print_printer": "Mozilla Save to PDF"
  },
  "media": {
    "currentAudioBackend": "pulse-rust",
    "currentMaxAudioChannels": 2,
    "currentPreferredSampleRate": 48000,
    "audioOutputDevices": [
      {
        "name": "USB Audio Digital Stereo (IEC958)",
        "groupId": "/devices/pci0000:40/0000:40:01.1/0000:41:00.0/0000:42:08.0/0000:46:00.3/usb7/7-5/7-5:1.0/sound/card3",
        "vendor": "Giga-Byte Technology Co., Ltd",
        "type": 2,
        "state": 2,
        "preferred": 15,
        "supportedFormat": 12336,
        "defaultFormat": 16,
        "maxChannels": 2,
        "defaultRate": 48000,
        "maxRate": 384000,
        "minRate": 1,
        "maxLatency": 0,
        "minLatency": 0
      },
      {
        "name": "Navi 10 HDMI Audio Digital Stereo (HDMI 3)",
        "groupId": "/devices/pci0000:40/0000:40:03.1/0000:49:00.0/0000:4a:00.0/0000:4b:00.1/sound/card2",
        "vendor": "Advanced Micro Devices, Inc. [AMD/ATI]",
        "type": 2,
        "state": 2,
        "preferred": 0,
        "supportedFormat": 12336,
        "defaultFormat": 16,
        "maxChannels": 2,
        "defaultRate": 44100,
        "maxRate": 384000,
        "minRate": 1,
        "maxLatency": 0,
        "minLatency": 0
      }
    ],
    "audioInputDevices": [
      {
        "name": "Monitor of USB Audio Digital Stereo (IEC958)",
        "groupId": "/devices/pci0000:40/0000:40:01.1/0000:41:00.0/0000:42:08.0/0000:46:00.3/usb7/7-5/7-5:1.0/sound/card3",
        "vendor": "Giga-Byte Technology Co., Ltd",
        "type": 1,
        "state": 2,
        "preferred": 15,
        "supportedFormat": 12336,
        "defaultFormat": 16,
        "maxChannels": 2,
        "defaultRate": 48000,
        "maxRate": 384000,
        "minRate": 1,
        "maxLatency": 0,
        "minLatency": 0
      },
      {
        "name": "Monitor of Navi 10 HDMI Audio Digital Stereo (HDMI 3)",
        "groupId": "/devices/pci0000:40/0000:40:03.1/0000:49:00.0/0000:4a:00.0/0000:4b:00.1/sound/card2",
        "vendor": "Advanced Micro Devices, Inc. [AMD/ATI]",
        "type": 1,
        "state": 2,
        "preferred": 0,
        "supportedFormat": 12336,
        "defaultFormat": 16,
        "maxChannels": 2,
        "defaultRate": 44100,
        "maxRate": 384000,
        "minRate": 1,
        "maxLatency": 0,
        "minLatency": 0
      }
    ]
  },
  "accessibility": {
    "isActive": false,
    "forceDisabled": 0,
    "handlerUsed": false,
    "instantiator": ""
  },
  "startupCache": {
    "DiskCachePath": "/home/amy/.cache/mozilla/firefox/v7vmcior.default/startupCache/startupCache.8.little",
    "IgnoreDiskCache": false,
    "FoundDiskCacheOnInit": false,
    "WroteToDiskCache": false
  },
  "libraryVersions": {
    "NSPR": {
      "minVersion": "4.32",
      "version": "4.32"
    },
    "NSS": {
      "minVersion": "3.71",
      "version": "3.71"
    },
    "NSSUTIL": {
      "minVersion": "3.71",
      "version": "3.71"
    },
    "NSSSSL": {
      "minVersion": "3.71",
      "version": "3.71"
    },
    "NSSSMIME": {
      "minVersion": "3.71",
      "version": "3.71"
    }
  },
  "userJS": {
    "exists": false
  },
  "intl": {
    "localeService": {
      "requested": [
        "en-US"
      ],
      "available": [
        "en-US"
      ],
      "supported": [
        "en-US"
      ],
      "regionalPrefs": [
        "en-IE"
      ],
      "defaultLocale": "en-US"
    },
    "osPrefs": {
      "systemLocales": [
        "en-IE"
      ],
      "regionalPrefsLocales": [
        "en-IE"
      ]
    }
  },
  "sandbox": {
    "hasSeccompBPF": true,
    "hasSeccompTSync": true,
    "hasPrivilegedUserNamespaces": true,
    "hasUserNamespaces": true,
    "canSandboxContent": true,
    "canSandboxMedia": true,
    "syscallLog": [],
    "contentSandboxLevel": 4,
    "effectiveContentSandboxLevel": 4,
    "contentWin32kLockdownState": "Win32k Lockdown disabled -- Operating system not supported"
  },
  "remoteAgent": {
    "listening": false,
    "url": ""
  },
  "graphics": {
    "numTotalWindows": 7,
    "numAcceleratedWindows": 7,
    "windowLayerManagerType": "WebRender (Software)",
    "windowLayerManagerRemote": true,
    "windowUsingAdvancedLayers": false,
    "adapterDescription": "AMD Radeon RX 5700 (NAVI10, DRM 3.40.0, 5.10.79, LLVM 13.0.0)",
    "adapterVendorID": "0x1002",
    "adapterDeviceID": "0x731f",
    "adapterRAM": 0,
    "adapterDrivers": "",
    "driverVendor": "mesa/radeonsi",
    "driverVersion": "21.2.5.0",
    "driverDate": "",
    "adapterVendorID2": "",
    "adapterDeviceID2": "",
    "isGPU2Active": false,
    "usesTiling": false,
    "targetFrameRate": 60,
    "windowProtocol": "wayland",
    "desktopEnvironment": "unknown",
    "webgl1Renderer": "WebGL is currently disabled.",
    "webgl1Version": "-",
    "webgl1DriverExtensions": "-",
    "webgl1Extensions": "-",
    "webgl1WSIInfo": "-",
    "webgl2Renderer": "WebGL is currently disabled.",
    "webgl2Version": "-",
    "webgl2DriverExtensions": "-",
    "webgl2Extensions": "-",
    "webgl2WSIInfo": "-",
    "info": {
      "AzureFallbackCanvasBackend": "skia",
      "AzureCanvasBackend": "skia",
      "AzureContentBackend": "skia",
      "ApzWheelInput": 1,
      "ApzDragInput": 1,
      "ApzKeyboardInput": 1,
      "ApzAutoscrollInput": 1,
      "ApzZoomingInput": 1,
      "CMSOutputProfile": "Empty profile data",
      "DisplayCount": 2,
      "Display0": "2560x1440 ",
      "Display1": "2560x1440 "
    },
    "featureLog": {
      "features": [
        {
          "name": "HW_COMPOSITING",
          "description": "Compositing",
          "status": "blocked",
          "log": [
            {
              "type": "default",
              "status": "available"
            },
            {
              "type": "runtime",
              "status": "blocked",
              "message": "Acceleration blocked by safe-mode"
            }
          ]
        },
        {
          "name": "OPENGL_COMPOSITING",
          "description": "OpenGL Compositing",
          "status": "unavailable",
          "log": [
            {
              "type": "default",
              "status": "unavailable",
              "message": "Hardware compositing is disabled"
            }
          ]
        },
        {
          "name": "WEBRENDER",
          "description": "WebRender",
          "status": "unavailable-in-safe-mode",
          "log": [
            {
              "type": "default",
              "status": "available"
            },
            {
              "type": "runtime",
              "status": "unavailable-in-safe-mode",
              "message": "Safe-mode is enabled"
            }
          ]
        },
        {
          "name": "WEBRENDER_QUALIFIED",
          "description": "WebRender qualified",
          "status": "available",
          "log": [
            {
              "type": "default",
              "status": "available"
            }
          ]
        },
        {
          "name": "WEBRENDER_COMPOSITOR",
          "description": "WebRender native compositor",
          "status": "blocklisted",
          "log": [
            {
              "type": "default",
              "status": "disabled",
              "message": "Disabled by default"
            },
            {
              "type": "env",
              "status": "blocklisted",
              "message": "Blocklisted by gfxInfo"
            }
          ]
        },
        {
          "name": "WEBRENDER_PARTIAL",
          "description": "WebRender partial present",
          "status": "available",
          "log": [
            {
              "type": "default",
              "status": "available"
            }
          ]
        },
        {
          "name": "WEBRENDER_SHADER_CACHE",
          "description": "WebRender shader disk cache",
          "status": "unavailable",
          "log": [
            {
              "type": "default",
              "status": "disabled",
              "message": "Disabled by default"
            },
            {
              "type": "runtime",
              "status": "unavailable",
              "message": "WebRender disabled"
            }
          ]
        },
        {
          "name": "WEBRENDER_OPTIMIZED_SHADERS",
          "description": "WebRender optimized shaders",
          "status": "unavailable",
          "log": [
            {
              "type": "default",
              "status": "available"
            },
            {
              "type": "runtime",
              "status": "unavailable",
              "message": "WebRender disabled"
            }
          ]
        },
        {
          "name": "WEBRENDER_ANGLE",
          "description": "WebRender ANGLE",
          "status": "unavailable",
          "log": [
            {
              "type": "default",
              "status": "available"
            },
            {
              "type": "env",
              "status": "unavailable",
              "message": "OS not supported"
            }
          ]
        },
        {
          "name": "WEBRENDER_DCOMP_PRESENT",
          "description": "WebRender DirectComposition",
          "status": "unavailable",
          "log": [
            {
              "type": "default",
              "status": "available"
            },
            {
              "type": "user",
              "status": "disabled",
              "message": "User disabled via pref"
            },
            {
              "type": "env",
              "status": "unavailable",
              "message": "Requires Windows 10 or later"
            },
            {
              "type": "runtime",
              "status": "unavailable",
              "message": "Requires ANGLE"
            }
          ]
        },
        {
          "name": "WEBRENDER_SOFTWARE",
          "description": "WebRender software fallback",
          "status": "available",
          "log": [
            {
              "type": "default",
              "status": "available"
            }
          ]
        },
        {
          "name": "WEBGPU",
          "description": "WebGPU",
          "status": "blocked",
          "log": [
            {
              "type": "default",
              "status": "disabled",
              "message": "Disabled by default"
            },
            {
              "type": "runtime",
              "status": "blocked",
              "message": "WebGPU can only be enabled in nightly"
            }
          ]
        },
        {
          "name": "X11_EGL",
          "description": "X11 EGL",
          "status": "available",
          "log": [
            {
              "type": "default",
              "status": "available"
            }
          ]
        },
        {
          "name": "DMABUF",
          "description": "DMABUF",
          "status": "available",
          "log": [
            {
              "type": "default",
              "status": "available"
            }
          ]
        }
      ],
      "fallbacks": []
    },
    "crashGuards": []
  },
  "experimentalFeatures": [
    [
      "experimental-features-abouthome-startup-cache",
      "browser.startup.homepage.abouthome_cache.enabled",
      false
    ],
    [
      "experimental-features-cookie-samesite-lax-by-default2",
      "network.cookie.sameSite.laxByDefault",
      false
    ],
    [
      "experimental-features-cookie-samesite-none-requires-secure2",
      "network.cookie.sameSite.noneRequiresSecure",
      false
    ],
    [
      "experimental-features-cookie-samesite-schemeful",
      "network.cookie.sameSite.schemeful",
      false
    ],
    [
      "experimental-features-css-cascade-layers",
      "layout.css.cascade-layers.enabled",
      false
    ],
    [
      "experimental-features-css-constructable-stylesheets",
      "layout.css.constructable-stylesheets.enabled",
      false
    ],
    [
      "experimental-features-css-masonry2",
      "layout.css.grid-template-masonry-value.enabled",
      false
    ],
    [
      "experimental-features-devtools-compatibility-panel",
      "devtools.inspector.compatibility.enabled",
      false
    ],
    [
      "experimental-features-devtools-execution-context-selector",
      "devtools.webconsole.input.context",
      false
    ],
    [
      "experimental-features-devtools-serviceworker-debugger-support",
      "devtools.debugger.features.windowless-service-workers",
      false
    ],
    [
      "experimental-features-fission",
      "fission.autostart",
      false
    ],
    [
      "experimental-features-media-jxl",
      "image.jxl.enabled",
      false
    ],
    [
      "experimental-features-multi-pip",
      "media.videocontrols.picture-in-picture.allow-multiple",
      true
    ],
    [
      "experimental-features-ime-search",
      "browser.urlbar.keepPanelOpenDuringImeComposition",
      false
    ],
    [
      "experimental-features-web-api-inputmode",
      "dom.forms.inputmode",
      false
    ],
    [
      "experimental-features-web-gpu2",
      "dom.webgpu.enabled",
      false
    ],
    [
      "experimental-features-webrtc-global-mute-toggles",
      "privacy.webrtc.globalMuteToggles",
      false
    ],
    [
      "experimental-features-win32k-lockdown",
      "security.sandbox.content.win32k-disable",
      false
    ]
  ],
  "addons": [
    {
      "name": "Amazon.co.uk",
      "type": "extension",
      "version": "1.9",
      "isActive": true,
      "id": "amazon@search.mozilla.org"
    },
    {
      "name": "Bing",
      "type": "extension",
      "version": "1.3",
      "isActive": true,
      "id": "bing@search.mozilla.org"
    },
    {
      "name": "DuckDuckGo",
      "type": "extension",
      "version": "1.1",
      "isActive": true,
      "id": "ddg@search.mozilla.org"
    },
    {
      "name": "eBay",
      "type": "extension",
      "version": "1.3",
      "isActive": true,
      "id": "ebay@search.mozilla.org"
    },
    {
      "name": "Google",
      "type": "extension",
      "version": "1.1",
      "isActive": true,
      "id": "google@search.mozilla.org"
    },
    {
      "name": "Wikipedia (en)",
      "type": "extension",
      "version": "1.1",
      "isActive": true,
      "id": "wikipedia@search.mozilla.org"
    },
    {
      "name": "Display Text Directly",
      "type": "extension",
      "version": "0.1.1",
      "isActive": false,
      "id": "display-text-directly@lukas-mai.addons.mozilla.org"
    },
    {
      "name": "DownThemAll!",
      "type": "extension",
      "version": "4.2.6",
      "isActive": false,
      "id": "{DDC359D1-844A-42a7-9AA1-88A850A938A8}"
    },
    {
      "name": "DuckDuckGo Privacy Essentials",
      "type": "extension",
      "version": "2021.9.30",
      "isActive": false,
      "id": "jid1-ZAdIEUB7XOzOJw@jetpack"
    },
    {
      "name": "Easy Youtube Video Downloader Express",
      "type": "extension",
      "version": "17.5",
      "isActive": false,
      "id": "{b9acf540-acba-11e1-8ccb-001fd0e08bd4}"
    },
    {
      "name": "Enhancer for YouTube™",
      "type": "extension",
      "version": "2.0.108",
      "isActive": false,
      "id": "enhancerforyoutube@maximerf.addons.mozilla.org"
    },
    {
      "name": "GitLab Markdown Viewer",
      "type": "extension",
      "version": "1.2.8",
      "isActive": false,
      "id": "{a8723c8a-1541-44c9-9a4f-2b55a994525d}"
    },
    {
      "name": "Markdown Viewer Webext",
      "type": "extension",
      "version": "1.8.0",
      "isActive": false,
      "id": "{943b8007-a895-44af-a672-4f4ea548c95f}"
    },
    {
      "name": "NoScript",
      "type": "extension",
      "version": "11.2.11",
      "isActive": false,
      "id": "{73a6fe31-595d-460b-a920-fcc0f8843232}"
    },
    {
      "name": "Open as Text",
      "type": "extension",
      "version": "1.2",
      "isActive": false,
      "id": "{45ed78de-8479-43f7-8cdf-1efdecd8ad2c}"
    },
    {
      "name": "Open in Browser",
      "type": "extension",
      "version": "2.11",
      "isActive": false,
      "id": "openinbrowser@www.spasche.net"
    },
    {
      "name": "Privacy Badger",
      "type": "extension",
      "version": "2021.8.17",
      "isActive": false,
      "id": "jid1-MnnxcxisBPnSXQ@jetpack"
    },
    {
      "name": "Terms of Service; Didn’t Read",
      "type": "extension",
      "version": "4.1.2",
      "isActive": false,
      "id": "jid0-3GUEt1r69sQNSrca5p8kx9Ezc3U@jetpack"
    },
    {
      "name": "Video Downloader professional",
      "type": "extension",
      "version": "1.99.20",
      "isActive": false,
      "id": "ffext_basicvideoext@startpage24"
    },
    {
      "name": "Video DownloadHelper",
      "type": "extension",
      "version": "7.6.0",
      "isActive": false,
      "id": "{b9db16a4-6edc-47ec-a1f4-b86292ed211d}"
    }
  ],
  "features": [
    {
      "name": "DoH Roll-Out",
      "version": "2.0.0",
      "id": "doh-rollout@mozilla.org"
    },
    {
      "name": "Firefox Screenshots",
      "version": "39.0.1",
      "id": "screenshots@mozilla.org"
    },
    {
      "name": "Form Autofill",
      "version": "1.0.1",
      "id": "formautofill@mozilla.org"
    },
    {
      "name": "Picture-In-Picture",
      "version": "1.0.0",
      "id": "pictureinpicture@mozilla.org"
    },
    {
      "name": "Proxy Failover",
      "version": "1.0.2",
      "id": "proxy-failover@mozilla.com"
    },
    {
      "name": "Web Compatibility Interventions",
      "version": "27.1.0",
      "id": "webcompat@mozilla.org"
    },
    {
      "name": "WebCompat Reporter",
      "version": "1.4.2",
      "id": "webcompat-reporter@mozilla.org"
    }
  ],
  "normandy": {
    "addonStudies": [],
    "prefRollouts": [],
    "prefStudies": [],
    "nimbusExperiments": [],
    "remoteConfigs": []
  }
}
```

**Comment by mweinelt** [comment]
Duplicate of https://github.com/NixOS/nixpkgs/issues/146382#issuecomment-974871842. Fixed in #146784.
